commcare_connect/form_receiver/processor.py

```diff
@@ -146,7 +147,7 @@ def process_deliver_unit(user, xform: XForm, app: CommCareApp, opportunity: Oppo
     access = OpportunityAccess.objects.get(opportunity=opportunity, user=user)
     counts = (
         UserVisit.objects.filter(opportunity=opportunity, user=user)
-        .exclude(status=VisitValidationStatus.over_limit)
+        .exclude(status=VisitValidationStatus.over_limit, is_trial=True)
```

Comment:
This needs to be changed to the below suggestion. The exclude method does an AND on the filter kwargs so this would cause only Visits that have both `VisitValidationStatus=over_limit AND is_trial=true` to be excluded and will have the wrong daily count.
```suggestion
        .exclude(Q(status=VisitValidationStatus.over_limit) | Q(is_trial=True))
```

Reply:
Right, updated.